### ISSUE TITLE
List order isn't preserved when opening a flag in the Reporting tab and then going back

### ISSUE THREAD

**Issue by vilanz** [Unleash/unleash-frontend#302]
I'm going through our old unused feature flags and this behavior is getting in the way.

#### Steps to reproduce

1. Open the Reporting tab
2. Click to a column to order the flags, e.g "Created"
3. The flags are now ordered by their creation date :)
4. Click on a specific flag to view it in detail
5. Go back to the Reporting tab with your browser's back button or whatever
6. The flags are not ordered by their creation date anymore :(

**Comment by vilanz** [comment]
I suppose adding the order column's name as an URL parameter in the router could fix this.
Something like https://unleash.herokuapp.com/reporting?sortBy=created

**Comment by FredrikOseberg** [comment]
I agree this would be nice to have. We're in the process of rethinking many parts of our UI, and reporting will also be touched upon in this process. I'll keep this in mind for when we revisit reporting in the near future.
